docs/elements/guides/sign-up.mdx

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to build a complete sign-up form with Clerk Elements.
+---
+
+# Build a sign-up flow
+
+In this guide you will learn how to build a complete sign-up flow using Clerk Elements components.
+
+<Callout type="info">
+
+Clerk Elements currently only works with Next.js and [Clerk Core 2](https://clerk.com/changelog/2024-04-19). As it gets closer to a stable release, support for additional frameworks will be added. If your Next.js application is already using Clerk, make sure to [upgrade to Core 2](/docs/upgrade-guides/core-2/nextjs). If you are starting from scratch, follow the [Next.js quickstart](/docs/quickstarts/nextjs) before proceeding.
```

Comment:
Please use the suggestion change I made for this in [the sign-in guide](https://github.com/clerk/clerk-docs/pull/890/files/3bd5d665cc7641c0d2db48655b79b84bbe48cec0#diff-9addefb83500d3a49204d58b6636b2020b87608132d4aec8e1451e79cacefa25R2) above